mobileapps/views.py

```diff
@@ -296,3 +300,206 @@ def delete(self, request, mobileapp_id):
         except ObjectDoesNotExist:
             raise Http404
 
+
+class MobileAppAllUsersNotifications(SecureListCreateAPIView):
+    """
+    **Use Cases**
+
+        send a push notification to all the users of the specific app
+        using urban airship push notification backend.
+
+    **Example Requests**
+
+        POST /api/mobileapps/{id}/notification
+
+        The body of the POST request must include the following parameters.
+
+        * title: notification title
+        * message: notification message
+
+    **Response Values**
+
+        If the request is successful, the request returns an HTTP 202 "Accepted" response.
+
+        The HTTP 202 response has the following value.
+
+        * message: Accepted
+    """
+
+    def post(self, request, pk):
+
+        message = request.data.get('message', None)
+        if not message:
+            return Response({'message': _('message is missing')}, status.HTTP_400_BAD_REQUEST)
+        title = request.data.get('title', None)
+        if not title:
+            return Response({'message': _('title is missing')}, status.HTTP_400_BAD_REQUEST)
+
+        try:
+            mobile_app = MobileApp.objects.get(pk=pk)
+            if not mobile_app.is_active:
+                return Response({'message': _('Mobile app is inactive')}, status.HTTP_403_FORBIDDEN)
+        except ObjectDoesNotExist:
+            return Response({'message': _('Mobile app does not exist')}, status.HTTP_404_NOT_FOUND)
+
+        try:
+            api_keys = get_api_keys(mobile_app)
+            notification_provider = get_provider_name(mobile_app)
+            if not notification_provider:
+                return Response({'message': _('Notification Provider not found')}, status.HTTP_404_NOT_FOUND)
+            notification_type = get_notification_type(u'open-edx.mobileapps.notifications')
+            notification_message = NotificationMessage(
+                namespace=mobile_app.identifier,
+                msg_type=notification_type,
+                payload={'title': title,
```

Comment:
set `title` key to message text and remove `message` key of payload here since it is not going to be used.